libs/zedUpload/awsutil/s3sync.go

```diff
@@ -263,6 +285,19 @@ func (s *S3ctx) DownloadFile(fname, bname, bkey string,
 		}
 	}
 
+	// if PartSize differ from saved clean doneParts
+	// we may hit this in case of S3PartSize change or from different type of datastore
+	if doneParts.PartSize != S3PartSize {
+		// log only if parts provided
+		if len(doneParts.Parts) > 0 {
+			log.Printf("DownloadFile: stored PartSize (%d) is different from expected (%d), assume DownloadedParts are broken",
```

Comment:
Is there also some place in zedUpload/awsutil where we can log when there is a retry of a part?

Reply:
I put it few lines below (after requesting of getNeededParts).